kf_lib_data_ingest/common/constants.py

```diff
@@ -26,49 +28,75 @@ class HASH:
 
 class SPECIMEN:
     class COMPOSITION:
-        XENOGRAFT = 'Xenograft Tissue'
-        EBVI = 'Epstein-Barr Virus Immortalized Cells'
-        TISSUE = 'Solid Tissue'
-        BLOOD = 'Blood'
-        PERIPHERAL_BLOOD = 'Peripheral Blood'
-        BUCCAL_SWAB = 'Buccal Cells'
+        BLOOD = 'Peripheral Whole Blood'
+        BONE = 'Bone'
         BONE_MARROW = 'Bone Marrow'
+        BUCCAL_SWAB = 'Buccal Cells'
+        EBVI = 'Epstein-Barr Virus Immortalized Cells'
+        FIBROBLASTS = 'Fibroblasts'
+        LINE = 'Derived Cell Line'
+        LYMPHOCYTES = 'Lymphocytes'
+        MNC = 'Mononuclear Cells'
+        PLASMA = 'Plasma'
         SALIVA = 'Saliva'
-        LINES = 'Cell Lines'
-        BONE = "Bone"
+        TISSUE = 'Solid Tissue'
+        XENOGRAFT = 'Xenograft Tissue'
 
-    class TISSUE_TYPE:
-        TUMOR = 'Tumor'
-        GERMLINE = 'Normal'
+    class SAMPLE_PROCUREMENT:
+        AUTOPSY = 'Autopsy'
+        BIOPSY = 'Biopsy'
+        BLOOD_DRAW = 'Blood Draw'
+        SUBTOTAL_RESECTIONS = 'Subtotal Resections'
+        TOTAL_RESECTIONS = 'Gross Total Resections'
 
     class ANATOMY_SITE:
+        BONE_MARROW = 'Bone Marrow'
+        HAIR = 'Hair'
         SKULL = 'Skull'
+        NERVOUS = 'Central Nervous System'
```

Comment:
```suggestion
        CNS = 'Central Nervous System'
```

Reply:
Thanks for the suggestion. I make that change.